src/util/net/fd_ip4.h

```diff
@@ -21,6 +21,16 @@
 #define FD_IP4_OPT_RA  ((uchar)148) /* This option is a router alert option */
 #define FD_IP4_OPT_EOL ((uchar)0)   /* This is the end of the options list */
 
+#define IP4_PRIVATE_RANGE1_START 0x0a000000 // 10.0.0.0
```

Comment:
Up to you but could be a little clearer to use `FD_IP4_ADDR(10,0,0,0)` here, will get optimized down anyway.